src/modules/home/WelcomeMessageSection.tsx

```diff
@@ -0,0 +1,82 @@
+import { InformationCircleIcon, XIcon } from '@heroicons/react/solid'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import * as React from 'react'
+
+const LOCAL_STORAGE_KEY = 'kominfod-welcome'
+const canUseDOM = typeof window !== 'undefined'
+const getLocalStorage = (key: string) => canUseDOM && localStorage.getItem(key)
+const setLocalStorage = (key: string, value: string) => {
+  return canUseDOM && localStorage.setItem(key, value)
+}
+
+export function WelcomeMessageSection() {
+  const router = useRouter()
+  const existingStorage = getLocalStorage(LOCAL_STORAGE_KEY)
+  const [isOpen, setIsOpen] = React.useState<boolean>(false)
+  const [hasStorage, setHasStorage] = React.useState<boolean>(
+    Boolean(existingStorage && existingStorage === '1')
+  )
+
+  const setCloseStateToStorage = () => {
+    setLocalStorage(LOCAL_STORAGE_KEY, '1')
+    setHasStorage(true)
+  }
+
+  const handleCloseMessage = React.useCallback(() => {
+    setIsOpen(false)
+    setCloseStateToStorage()
+  }, [])
+
+  React.useEffect(() => {
+    // Only open the dialog when intersecting
+    // and have not been triggered before
+    // and doesn't have a local storage "wbw-wm"
+    if (!hasStorage) {
+      setIsOpen(true)
+    }
+  }, [hasStorage])
+
+  React.useEffect(() => {
+    // Also close dialog when route changes
+    router.events.on('beforeHistoryChange', handleCloseMessage)
+
+    return () => {
+      router.events.off('beforeHistoryChange', handleCloseMessage)
+    }
+  }, [router.events, handleCloseMessage])
+
+  if (isOpen) {
+    return (
+      <div className='rounded-md bg-brand-light-blue p-4'>
+        <div className='flex'>
+          <div className='flex-shrink-0'>
+            <InformationCircleIcon className='h-5 w-5 text-brand-blue' aria-hidden='true' />
```

Comment:
Artistic choice, I'm using `react-feather` / Feather Icons for icons already for the rest of the whole site. Do you think we should use another icon from a different icon set?